test/vignette/http/integration_passthrough_test.clj

```diff
@@ -0,0 +1,59 @@
+(ns vignette.http.integration-passthrough-test
```

Comment:
I had to put it in separate file because:
```
Exception in thread "main" java.lang.ClassFormatError: Invalid method Code length 66318 in class file vignette/http/integration_test$eval10458, compiling:(vignette/http/integration_test.clj:22:1)
```